src/Persistence_SQL.php

```diff
@@ -657,7 +662,7 @@ public function load(Model $m, $id)
      */
     public function tryLoadAny(Model $m)
     {
-        $load = $this->action($m, 'select');
+        $load = $m->action('select');
```

Comment:
except that it allows model to override actions, which is what union model does.